src/gluonts/time_feature/lag.py

```diff
@@ -111,16 +117,24 @@ def _make_lags_for_month(multiple, num_cycles=3):
     elif offset_name == "H":
         lags = (
             _make_lags_for_hour(offset.n)
-            + _make_lags_for_day(offset.n / 24.0)
-            + _make_lags_for_week(offset.n / (24.0 * 7))
+            + _make_lags_for_day(offset.n / 24)
+            + _make_lags_for_week(offset.n / (24 * 7))
         )
     # minutes
     elif offset_name == "T":
         lags = (
             _make_lags_for_minute(offset.n)
-            + _make_lags_for_hour(offset.n / 60.0)
-            + _make_lags_for_day(offset.n / (60.0 * 24))
-            + _make_lags_for_week(offset.n / (60.0 * 24 * 7))
+            + _make_lags_for_hour(offset.n / 60)
+            + _make_lags_for_day(offset.n / (60 * 24))
+            + _make_lags_for_week(offset.n / (60 * 24 * 7))
+        )
+    # second
+    elif offset_name == "S":
+        lags = (
+            _make_lags_for_second(offset.n)
+            + _make_lags_for_minute(offset.n / 60)
+            + _make_lags_for_hour(offset.n / (60 * 60))
+            + _make_lags_for_day(offset.n / (60 * 60 * 24))
```

Comment:
I think we can even remove `_make_lags_for_day` since it goes back 4 weeks in time, which is a lot, about 2.5 million time-steps.

`_make_lags_for_hour` max lagged value is going back `7` days, which should be plenty.